content/llm/all.md

```
@@ -1888,286 +1893,1698 @@ For security, always validate the API key in the `x-meeting-baas-api-key` header
+        <h4 className="font-medium">Specialty Roles</h4>
+        <ul className="list-disc pl-4">
+          <li>Academic Warlord</li>
+          <li>Climate Engineer</li>
+          <li>Deep Sea Therapist</li>
+          <li>Futuristic AI Philosopher</li>
+          <li>Military Strategist</li>
+          <li>Quantum Physicist</li>
+        </ul>
+      </div>
+    </div>
+    
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 4

(MD009, no-trailing-spaces)